website/README.md

```diff
@@ -18,14 +18,26 @@ The Beam website is built using [Jekyll](http://jekyllrb.com/). Additionally,
 for additional formatting capabilities, this website uses
 [Twitter Bootstrap](http://getbootstrap.com/).
 
-### Repository Structure
+Documentation generated from source code, such as Javadoc and Pydoc, is stored
+separately on the [beam-site
+repository](https://github.com/apache/beam-site/tree/asf-site/content/documentation/sdks).
 
-This repository contains:
+## Development Workflow with Docker
 
-1. `src/`: the source of the site, including markdown files containing the bulk of the content
-1. `content/`: html generated from the markdown (which is what is actually hosted on the website)
+### Active development
+
+If you have Docker configured on your machine, the following command may be used
+to build and serve the website locally.
+
+    $ ./gradlew :beam-website:serveWebsite
+
+Any changes made locally will trigger a rebuild of the website.
 
-## Development Workflow
+You can also run website tests using this command:
+
+    $ ./gradlew :beam-website:testWebsite
+
+## Development Workflow without Docker
 
 ### Setup
```

Comment:
Are these setup steps still necessary? I believe the website build all happens within a Docker container now.

Also, do you think we should continue to document the raw bundle / jekyll commands, or recommend that users only build/test via the gradle wrapper? My preference would be to only use gradle so we can change what's underneath more easily.

Reply:
Some of it may be of interest for maintaining the docker image itself, but should be moved out of the way for regular web site contributor.